lib/vrl/stdlib/src/to_timestamp.rs

```diff
@@ -163,7 +163,9 @@ mod tests {
         let mut object: Value = BTreeMap::new().into();
         let mut runtime_state = vrl::state::Runtime::default();
         let tz = TimeZone::default();
-        let mut ctx = Context::new(&mut object, &mut runtime_state, &tz);
+        let enrichment_tables =
+            Some(Box::new(vrl::EmptyEnrichmentTables) as Box<dyn vrl::EnrichmentTableSearch>);
+        let mut ctx = Context::new(&mut object, &mut runtime_state, &tz, &enrichment_tables);
```

Comment:
Any reason why we can't pass in `None` here? That would be more idiomatic, and simplifies the code here and in other places.

Reply:
None whatsoever!